scripts/inspector-findings.py

```diff
@@ -45,52 +48,59 @@ class ParseInspectorFindings:
     def _parse_args(cls, argv):
         import argparse
         parser = argparse.ArgumentParser(description=__doc__,
-                                         formatter_class=argparse.RawTextHelpFormatter)
-        parser.add_argument('--severity', '-s',
-                            default=','.join(cls.default_severities),
-                            help='Limit the results to the severities specified, '
-                                 'comma-separated with no spaces.\n'
-                                 f"Default: '{','.join(cls.default_severities)}'\n"
-                                 f"Choices: {', '.join(cls.all_severities)}")
-        parser.add_argument('--json', '-j',
+                                         formatter_class=AzulArgumentHelpFormatter)
+        parser.add_argument('--severity',
+                            nargs='+',
+                            metavar='S',
+                            default=cls.default_severities,
+                            choices={s: s for s in cls.all_severities},
+                            help='Only fetch vulnerabilities with the specified'
+                                 ' severity. '
+                                 f"(choices: {cls.all_severities})")
+        parser.add_argument('--json',
                             default=False, action='store_true',
-                            help='Dump all findings to a JSON file.')
+                            help='Create a JSON file export of the findings.')
```

Comment:
Nitpick, but I preferred the original wording.

```suggestion
                            help='Dump findings to a JSON file.')
```